haskell/ghc.BUILD

```diff
@@ -9,7 +9,7 @@ filegroup(
 
 cc_library(
   name = "threaded-rts",
-  srcs = glob(["lib/ghc-*/rts/libHSrts_thr-ghc*.so"]),
+  srcs = glob(["lib/ghc-*/rts/libHSrts_thr-ghc*. + ext for ext in ["so", "dylib"]]),
```

Comment:
There is a syntax error here. You probably forgot `"` after the `-ghc*.` part. Can you fix this and see if GHC bindist works for you on MacOS after that?

Reply:
FYI I hadn't pushed a fix to this yet since it was a little harder to test.  I can send another PR as a follow-up (also with updated links to the bindists).

Reply:
OK. Meanwhile I'm working on CI.